lib/Db/RefreshToken.php

```diff
@@ -33,6 +33,8 @@
  */
 class RefreshToken extends Entity {
 
+	const EXPIRATION_DAYS = 30;
```

Comment:
We should pull this from IConfig so that it can be set with `occ` and stored in `oc_appconfig`. So we should probably move this code to the Controller for now, and avoid the login / depdancy in here.